src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -184,23 +183,6 @@ public GameService(ClientProperties clientProperties, FafService fafService,
     games = FXCollections.observableList(new ArrayList<>(),
         item -> new Observable[]{item.statusProperty(), item.getTeams()}
     );
-    games.addListener((ListChangeListener<Game>) change -> {
```

Comment:
This is problematic because when updating teams of games.
1.first the teams are cleared
2. Everything is added back in
Other code relying on the Events do then think all players rejoined the games which is wrong so we better post the events in a different spot, there where we process the server message.